### DISCUSSION TITLE
Duplicate issue

### DISCUSSION THREAD

**Discussion by GogoFC** [immich-app/immich#11561] [Q&A]
### The bug

I hit Deduplicate all Photos to delete duplicates. There was about 1500 of them.  

Then in the trash I saw that some files were 1.5M and not 740kb and then I wasn't sure those were correct.

I hit Restore All to undelete all from trash. I had to do that couple of times and refresh, it said it only restored about 60 photos each time. So there's nothing in trash now. 

Now it shows there's about 160 duplicates in Utilities, where as there was 1500 a second ago.

The duplicates don't show as two photos side by side anymore, it shows only one photo.

Before running this I run immich-go duplicates but that didn't remove these 1500 duplicates, it did remove something else. This is after I imported google photos, not sure if this is relevant. 

![image](https://github.com/user-attachments/assets/a82e2ee9-c3ee-4544-ac09-d14ee70c2b25)

### The OS that Immich Server is running on

Ubuntu 24.04 LTS

### Version of Immich Server

1.111.0

### Version of Immich Mobile App

1.111.0 build.152

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
default
```

### Your .env content

```Shell
default
```

### Reproduction steps

```bash
1. Deduplicate All
2. Restore All from Trash
```

**Comment by bo0tzz** [comment]
Try rerunning the duplicate detection job.

**Reply by GogoFC** [reply]
Thanks I'll try that. Did not do that, I'm new to immich so I don't know where everything is and what's possible, like running the dd job.

**Reply by wit-go** [reply]
How do you rerun that? From the command line? I can't find it in the interface.

It's nerve racking to delete duplicates from the trash. I had 5000 after I imported everything I owned. An awesome feature in the trash would be to add a button for "delete duplicates" that specifically does that. Another great option would be to visually identify which images are in the trash because they were tagged as duplicates.

One has to have faith that the images in the trash are actually duplicates and not that I finger borked them into trash.

This project rocks.

**Reply by bo0tzz** [reply]
https://my.immich.app/admin/jobs-status
